src/Umbraco.Web.UI.Client/package.json

```diff
@@ -76,17 +76,17 @@
     "gulp-watch": "5.0.1",
     "gulp-wrap": "0.15.0",
     "gulp-wrap-js": "0.4.1",
-    "jasmine-core": "4.1.0",
-    "jsdom": "19.0.0",
-    "karma": "6.3.19",
-    "karma-jasmine": "5.0.0",
-    "karma-jsdom-launcher": "12.0.0",
+    "jasmine-core": "4.4.0",
+    "jsdom": "20.0.0",
```

Comment:
Can vi update dependencies to new major in a minor release? Or does this constitute a breaking change? I'm not sure what the rules are for the frontend